src/torchcodec/_core/ops.py

```diff
@@ -83,6 +83,11 @@ def load_torchcodec_shared_libraries() -> tuple[int, str]:
              table:
              https://github.com/pytorch/torchcodec?tab=readme-ov-file#installing-torchcodec.
           3. Another runtime dependency; see exceptions below.
+
+        Note: We attempt to load libtorchcodec for FFmpeg versions 8 down to 5.
```

Comment:
Let's incorporate this in point 1 above, since it's related. That would also make it so that we don't need to restate the supported versions.